sci-libs/cantera/cantera-2.4.0.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )
+
+inherit desktop python-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="+cpp +cti fortran pch python static-libs test"
+
+## Python2 is required by '<scons-3.0' to work.
+## Python3 automatic detection is used by cantera before compilling
+## if python3_package isn't set no "n".
+
+REQUIRED_USE="
+	|| ( cpp python )
+	cti? ( ${PYTHON_REQUIRED_USE} )
+	fortran? ( cpp )
+	python? ( cti )
+	static-libs? ( cpp )
+	?? ( python_targets_python3_4 python_targets_python3_5 python_targets_python3_6 python_targets_python3_7 )
+	"
+
+RDEPEND="
+	python? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+"
+
+# SCons >= 3.0 is recomended, but builds with older versions
+DEPEND="
+	${RDEPEND}
+	dev-cpp/eigen
+	dev-libs/boost
+	dev-libs/libfmt:0=
+	dev-util/scons
+	sci-libs/sundials
+	fortran? (
+		sci-libs/sundials[fortran]
+		sys-devel/gcc[fortran]
+	)
+	python? (
+		dev-python/3to2
+		dev-python/cython[${PYTHON_USEDEP}]
+	)
+	test? (
+		>=dev-cpp/gtest-1.8.0
+	)
+"
+
+PATCHES=( "${FILESDIR}/${PN}_${PV}_disable_debug_and_optimization.patch" )
+
+## Full list of configuration options of Cantera is presented here:
+## http://cantera.org/docs/sphinx/html/compiling/config-options.html
+
+scons_vars=()
```

Comment:
Why are you declaring it as global variable?

Reply:
I don't remember where I notice similar code but yes I definitely must to rework it and place assignment of SCons build parameters into appropriate ebuild sections. I will try to address the shortcomings (and other remarks) during this weekend.

Reply:
Done.